samples/client/src/main/java/global/covesa/sdk/client/MainViewModel.kt

```diff
@@ -27,6 +32,9 @@ class MainViewModel(
     var installedServicesUiState by mutableStateOf(InstalledServicesUiState())
         private set
 
+    var pushUiState by mutableStateOf(context?.let { PushUiState(it) } ?: PushUiState())
```

Comment:
UiState should not be dependent on `context`. Is it just initial value? You can get value of `PushManager.getAckDistributor(context)` in `Activity` and pass it to the `MainViewModel` constructor instead.